packages/flutter/test/material/calendar_date_picker_test.dart

```diff
@@ -657,7 +657,6 @@ void main() {
     group('Semantics', () {
       testWidgets('day mode', (WidgetTester tester) async {
         final SemanticsHandle semantics = tester.ensureSemantics();
-        addTearDown(semantics.dispose);
```

Comment:
This is surprising. I always thought this was the better way to dispose resource

Reply:
Yeah, but the way our checks currently work they don't play nice with that. We should probably fix that at some point.